src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package service.impl;
+
+import java.util.Map;
+import java.util.Set;
+import service.ReportService;
+import storage.ReportStorage;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String DATA_SEPARATOR = ",";
+    private static final String REPORT_HEADER = "fruit,quantity";
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final Set<Map.Entry<String, Integer>> ENTRIES =
+            new DataStorageServiceImpl().getFruitMap().entrySet();
+    private ReportStorage reportStorage;
+
+    @Override
+    public String getReport() {
+        makeReport();
+        return reportStorage.getReport();
+    }
```

Comment:
```suggestion
```
I don't see a reason to have this method + ReportStorage, if you have a perfectly working makeReport method that returns String report

Reply:
That left from alfa version maybe)
I was playing with storage services and direct accessing and so on.
Thanks.